### PR TITLE
Fixed #29461 -- Fixed ogrinspect test_time_field failure on SpatiaLite.

### PR DESCRIPTION
**Pull request by timgraham**
https://code.djangoproject.com/ticket/29461

### CODE REVIEW

**Comment by timgraham** [comment]
buildbot, test on on bionic.

**Comment by timgraham** [comment]
buildbot, test on bionic.

**Comment by timgraham** [comment]
buildbot, test on bionic.

File: tests/gis_tests/inspectapp/tests.py [hunk 1]
```diff
@@ -134,12 +134,20 @@ def test_time_field(self):
         ))
 
         # The ordering of model fields might vary depending on several factors (version of GDAL, etc.)
-        self.assertIn('    f_decimal = models.DecimalField(max_digits=0, decimal_places=0)', model_def)
+        if connection.vendor == 'sqlite':
+            # SpatiaLite introspection is somewhat lacking (#29461).
+            self.assertIn('    f_decimal = models.CharField(max_length=0)', model_def)
```
Comment (claudep) [inline tests/gis_tests/inspectapp/tests.py]: I'm not sure I would even write an assertion for such a broken output. Just ignore? Same below.

Comment (timgraham) [reply tests/gis_tests/inspectapp/tests.py]: An advantage could be that we would know if SpatiaLite's behavior changes.

Comment (claudep) [reply tests/gis_tests/inspectapp/tests.py]: Makes sense, go ahead!

**Pull request merged**